superset.gemspec

```diff
@@ -35,20 +35,20 @@ Gem::Specification.new do |spec|
   ]
 
   # Uncomment to register a new dependency of your gem
-  spec.add_dependency "dotenv", "~> 2.7"
-  spec.add_dependency "json", "~> 2.6"
+  spec.add_dependency "dotenv", ">= 2.0"
+  spec.add_dependency "json", ">= 2.0"
   spec.add_dependency "terminal-table", "~> 4.0"
-  spec.add_dependency "rake", "~> 13.0"
-  spec.add_dependency "rollbar", "~> 3.4"
-  spec.add_dependency "require_all", "~> 3.0"
-  spec.add_dependency "rubyzip", "~> 1.0"
+  spec.add_dependency "rake", ">= 13.0"
+  spec.add_dependency "rollbar", ">= 3.0"
+  spec.add_dependency "require_all", ">= 3.0"
+  spec.add_dependency "rubyzip", ">= 1.3"
   spec.add_dependency "faraday", "~> 1.0"
```

Comment:
Add some point we'll need to look into migrating our apps to faraday v2

Reply:
Agreed .. some work on Happi gem started on that a few months ago, wip
https://github.com/rdytech/happi/releases. rc1

and
https://github.com/rdytech/happi/pull/21 Rc2